projects/appearances/protos/Soil.proto

```diff
@@ -0,0 +1,38 @@
+#VRML_SIM R2019b utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# tags: static
+# A soil material.
+
+PROTO Soil [
+  field SFString{"braun", "grey"}  soilColor         "braun"  # Defines the color of the soil
```

Comment:
I like the idea to merge several materials thanks to an enumeration. However, I don't think `soilColor` is a good way to refer them. Why not simply a `type` field with a text explaining what it is like `rocky grey dry mud, etc.`.? It would seem more scalable to me.